pytest.ini

```diff
@@ -1,2 +1,4 @@
 [pytest]
-addopts = --doctest-modules
\ No newline at end of file
+addopts = --doctest-modules
+filterwarnings = 
```

Comment:
This configuration will suppress `pytest` with  `travis` to capture or show `optimizewarning`.  It is very weird that when testing locally with `pytest`, it won't fail but to give a warning, while when building with `travis`, `pytest` will always fail and exit with code 1.
![image](https://user-images.githubusercontent.com/7359284/39774046-a142af40-52ae-11e8-9090-a66b9470ecb1.png)